modules/priority-expander-autoscaler-hcp.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-machine-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="priority-expander-autoscaler-hcp_{context}"]
+= Setting the priority expander in a hosted cluster
+
+You can define the priority for your node pools and create high priority machines before low priority machines by using the priority expander in your hosted cluster.
+
+.Prerequisites
+
+* You have created the `HostedCluster` and `NodePool` resources.
+
+.Procedure
+
+. To define the priority for your node pools, create a config map named `priority-expander-configmap.yaml` in your hosted cluster. Node pools with low numbers receive high priority. See the following example configuration:
++
+.Example `priority-expander-configmap.yaml` file
```

Comment:
The AsciiDita migration tool might flag these code block headings. Consider building the heading text into the lead-in paragraph of the step.